.travis.yml

```diff
@@ -1,13 +1,13 @@
 ---
 sudo: required
-dist: trusty
+dist: bionic
 
 language: c
 
 before_install:
   - sudo apt-get update -q
   - sudo apt-get install -qq apt-transport-https python-docutils
-  - curl -s https://packagecloud.io/install/repositories/varnishcache/varnish41/script.deb.sh | sudo bash
+  - curl -s https://packagecloud.io/install/repositories/varnishcache/varnish60/script.deb.sh | sudo bash
```

Comment:
we actually use https://packagecloud.io/varnishcache/varnish60lts for this one

Reply:
ah right - done :)